lesson_11/Unit_test.py

```diff
@@ -0,0 +1,44 @@
+from homework_11 import log_event
+import unittest
+
+class TestHW11(unittest.TestCase):
+
+    def test_01(self):
+        """Тест успішного входу"""
+        try:
+            log_event("User123", "success")
+            result = True
+        except Exception:
+            result = False
+        self.assertTrue(result)
```

Comment:
```suggestion
        try:
            log_event("User123", "success")
            result = True
        except Exception:
            result = False
        self.assertTrue(result)
```
ясно, що нічого не понятно.
ви передаєте дані у лог. І задаєт резалт - тру. 
Ну так тут як такого тесту немає. Ви ж задали результат
З такім же успіхом можна було написати 
`self.assertTrue(True)`
Лінь це добре, але злість повинна ставати їй на заваді,